conda-store-server/conda_store_server/schema.py

```diff
@@ -615,6 +620,28 @@ class APIGetNamespace(APIResponse):
     data: Namespace
 
 
+# GET /api/v1/namespace/{name}/role
+class APIGetNamespaceRole(BaseModel):
+    other_namespace: str
```

Comment:
It's a bit strange to pass JSON for a GET, but I wanted to have similar interfaces everywhere. And I don't think it's forbidden by REST. I also didn't want to have too many parameters in the URL itself. I'm happy to reconsider if people feel strongly about this and can provide arguments against this.

Reply:
I guess one argument might be is that it'd be easier to see the data in a browser if you don't have to pass external params.

Reply:
It's common to use query parameters for GET requests. Could this be used? other_namespace is optional. And depending on if it is specified the scope of `GetNamespaceRoles` will return fewer results.

Reply:
For other reviewers: I'll try using query params here. Pros: easier to test in a browser + caching.